### PR TITLE
[ZEPPELIN-3957] Divide SparkIntegrationTest into parts for build under 50 minutes

### PR DESCRIPTION
**Pull request by D01B**
### What is this PR for?
Fix Travis profiles and tests to archive build time under 50 minutes. 

### What type of PR is it?
[Improvement]

### What is the Jira issue?
*  [ZEPPELIN-3957]

### How should this be tested?
* Travis CI pass

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by D01B** [comment]
@felixcheung, could you review, please?

**Comment by zjffdu** [comment]
Thanks @D01B  +1

File: .travis.yml [hunk 1]
```diff
@@ -92,11 +92,17 @@ matrix:
...
 
-    # Integration test of spark interpreter with different spark versions under python3, only run SparkIntegrationTest. Also run spark unit test of spark 1.6 in this build.
+    # Integration test of spark interpreter with different spark versions under python3, only run SparkIntegrationTestPt1. Also run spark unit test of spark 1.6 in this build.
     - sudo: required
       jdk: "oraclejdk8"
       dist: trusty
-      env: BUILD_PLUGINS="true" PYTHON="3" SCALA_VER="2.10" PROFILE="-Pspark-1.6 -Pscala-2.10" SPARKR="true" BUILD_FLAG="install -DskipTests -DskipRat -am" TEST_FLAG="test -DskipRat -am" MODULES="-pl zeppelin-zengine,spark/interpreter,spark/spark-dependencies" TEST_PROJECTS="-Dtest=SparkIntegrationTest,org.apache.zeppelin.spark.* -DfailIfNoTests=false"
+      env: BUILD_PLUGINS="true" PYTHON="3" SCALA_VER="2.10" PROFILE="-Pspark-1.6 -Pscala-2.10" SPARKR="true" BUILD_FLAG="install -DskipTests -DskipRat -am  " TEST_FLAG="test -DskipRat -am" MODULES="-pl zeppelin-zengine,spark/interpreter,spark/spark-dependencies" TEST_PROJECTS="-Dtest=SparkIntegrationTestPt1,org.apache.zeppelin.spark.* -DfailIfNoTests=false"
```
Comment (felixcheung) [inline .travis.yml]: this change `-am  " TEST_FLAG` vs `-am" TEST_FLAG` is it intentional?

File: zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/SparkIntegrationTestPt1.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  public SparkIntegrationTestPt1(String sparkVersion) {
+    super(sparkVersion);
+  }
+
+  @Parameterized.Parameters
+  public static List<Object[]> data() {
+    return Arrays.asList(new Object[][]{
+            {"2.4.0"},
+            {"2.3.2"},
```
Comment (felixcheung) [inline zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/SparkIntegrationTestPt1.java]: looks like we need to upgrade to 2.3.3 (soon) and 2.2.3..

File: zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/SparkIntegrationTestPt2.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+public class SparkIntegrationTestPt2 extends SparkIntegrationTest{
+
+  public SparkIntegrationTestPt2(String sparkVersion) {
+    super(sparkVersion);
+  }
+
+  @Parameterized.Parameters
+  public static List<Object[]> data() {
+    return Arrays.asList(new Object[][]{
+            {"2.1.2"},
```
Comment (felixcheung) [inline zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/SparkIntegrationTestPt2.java]: ditto 2.1.3

**Comment by felixcheung** [comment]
merging if no more comment

File: .travis.yml (hunk 1 above)
Comment (D01B) [reply .travis.yml]: oh, additional space is not intentional, it's my mistake, I can fix this, but I think it doesn't matter because of cmd parsing rules

Comment (felixcheung) [reply .travis.yml]: ok, if it's easy could you revert that? also same in L105

**Comment by D01B** [comment]
@felixcheung I think in the next couple months we may upgrade the spark version, in this month we don't have enough time.

File: .travis.yml (hunk 1 above)
Comment (D01B) [reply .travis.yml]: ok, I'll fix this

File: .travis.yml [hunk 4]
```diff
@@ -92,11 +92,17 @@ matrix:
...
+      env: BUILD_PLUGINS="true" PYTHON="3" SCALA_VER="2.10" PROFILE="-Pspark-1.6 -Pscala-2.10" SPARKR="true" BUILD_FLAG="install -DskipTests -DskipRat -am" TEST_FLAG="test -DskipRat -am" MODULES="-pl zeppelin-zengine,spark/interpreter,spark/spark-dependencies" TEST_PROJECTS="-Dtest=SparkIntegrationTestPt1,org.apache.zeppelin.spark.* -DfailIfNoTests=false"
+
+    # Integration test of spark interpreter with different spark versions under python3, only run SparkIntegrationTestPt2. Also run spark unit test of spark 1.6 in this build.
+    - sudo: required
+      jdk: "oraclejdk8"
+      dist: trusty
+      env: BUILD_PLUGINS="true" PYTHON="3" SCALA_VER="2.10" PROFILE="-Pspark-1.6 -Pscala-2.10" SPARKR="true" BUILD_FLAG="install -DskipTests -DskipRat -am" TEST_FLAG="test -DskipRat -am" MODULES="-pl zeppelin-zengine,spark/interpreter,spark/spark-dependencies" TEST_PROJECTS="-Dtest=SparkIntegrationTestPt2,org.apache.zeppelin.spark.* -DfailIfNoTests=false"
```
Comment (D01B) [inline .travis.yml]: @felixcheung, I fix this(remove additional space) L99, L105

**Pull request merged**